Lib/base64.py

```diff
@@ -570,8 +570,7 @@ def main():
     usage = """usage: %s [-h|-d|-e|-u|-t] [file|-]
```

Comment:
nit: Can we update `usage` to use f-string?

Reply:
Done. Should I go further and convert it to argparse?